controllers/rcj_soccer_referee_supervisor/referee/consts.py

```diff
@@ -1,12 +1,12 @@
 from enum import Enum
-
+import os
 
 class Team(Enum):
     BLUE = "B"
     YELLOW = "Y"
 
 
-MATCH_TIME = 10 * 60  # 10 minutes
+MATCH_TIME = os.environ.get("MATCH_TIME", 10 * 60)  # 10 minutes unless otherwise specified
```

Comment:
This is not good. `consts` are ment to be constants -- no changes from outside.
So, let's call this `DEFAULT_MATCH_TIME = 10 * 60`, extract `MATCH_TIME` in main file and if the env variable does not contain anything, then fall back to `DEFAULT_MATCH_TIME` and pass it to `Referee` instance via `match_time` argument.

Reply:
Fixed.

Reply:
Oops. My bad, didn't realize this would immediately materialize on the PR. I was just testing and pushed just to move the code to the other machine. I'll fix it before the next push.